src/IndicatorsDash.jsx

```diff
@@ -23,17 +22,15 @@ class IndicatorsDash extends React.Component {
       connError: false,
       data: [],
       activeTab: 'all',
-      selectedOption: null,
-      biomesByBlockData: [],
     };
   }
 
   componentDidMount() {
     const { activeArea } = this.props;
-
-    const areaId = (activeArea && activeArea.id) ? activeArea.id : 'LLA 0970';
-    this.loadIndicators(areaId);
-    this.loadBiomes(areaId);
+    const areaId = (activeArea && activeArea.name) ? activeArea.name : null;
+    if (areaId) {
+      this.loadIndicators(areaId);
+    }
```

Comment:
Here you definitively need the else statement to set connError to true

Reply:
Done